cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/ACLineSegmentConversion.java

```diff
@@ -41,27 +43,51 @@ public boolean valid() {
 
     @Override
     public void convert() {
+        convertLine();
+    }
+
+    @Override
+    public void convertAtBoundary() {
         if (isBoundary(1)) {
             convertLineAtBoundary(1);
         } else if (isBoundary(2)) {
             convertLineAtBoundary(2);
         } else {
-            convertLine();
+            throw new ConversionException("Boundary must be at one end of the line");
         }
     }
 
-    private void convertLineAtBoundary(int boundarySide) {
-        // If we have created buses and substations for boundary nodes,
-        // convert as a regular line
-        if (context.config().convertBoundary()) {
-            convertLine();
+    @Override
+    public BoundaryLine asBoundaryLine(String boundaryNode) {
+        BoundaryLine boundaryLine = super.createBoundaryLine(boundaryNode);
+        double r = p.asDouble("r");
+        double x = p.asDouble("x");
+        double g = p.asDouble("gch", 0);
+        double b = p.asDouble("bch", 0);
+        boundaryLine.setParameters(r, x, g / 2, b / 2, g / 2, b / 2);
+        return boundaryLine;
+    }
+
+    public static void convertBoundaryLines(Context context, String boundaryNode, BoundaryLine boundaryLine1, BoundaryLine boundaryLine2) {
+        Line mline;
+        if (context.config().mergeBoundariesUsingTieLines()) {
+            mline = createTieLine(context, boundaryNode, boundaryLine1, boundaryLine2);
         } else {
-            double r = p.asDouble("r");
-            double x = p.asDouble("x");
-            double gch = p.asDouble("gch", 0.0);
-            double bch = p.asDouble("bch");
+            mline = createLine(context, boundaryLine1, boundaryLine2);
+        }
```

Comment:
Is there a reason why we wouldn't want tie lines? I would convert them as tie lines in any case (I know we made the distinction before but I don't think it is necessary)

Reply:
I will remove it in a separated PR after merging this one.